README.md

```diff
@@ -374,15 +374,18 @@ TODO
 
 ### `scaling`
 
-TODO
+A simple OpenShift test verifying that the OpenShift deployment with Quarkus application scales up and down.
 
 ### `external-applications/todo-demo-app`
 
-TODO
+This test produces a S2I source deployment config for OpenShift with todo-demo-app serving a simple todo checklist.
+The test verifies that the application with a sample of libraries is buildable and deployable via supported means.
 
 ### `external-applications/quarkus-workshop-super-heroes`
 
-TODO
+This test produces a S2I source deployment config for OpenShift with Quarkus Super heroes workshop application.
+The test verifies that the application is buildable and deployable. It also verifies that the REST and MicroProfile APIs
+function properly on OpenShift, as well as the database integrations.
 
```

Comment:
State explicitly that source code of todo-demo-app and quarkus-workshop-super-heroes lives outside quarkus-openshift-test-suite.

Reply:
+1.

Also, "_an_ S2I source ..."